packages/wallet/backend/src/user/service.ts

```diff
@@ -72,7 +72,7 @@ export class UserService implements IUserService {
 
     const resetToken = getRandomToken()
     const passwordResetToken = hashToken(resetToken)
-    const passwordResetExpiresAt = new Date(Date.now() + 10 * 60 * 1000)
+    const passwordResetExpiresAt = new Date(Date.now() + 12 * 3600 * 1000)
```

Comment:
Set the hours to 8 hours, and update the email someweher to mention the expiry time. 
'Password reset link will expire in 8 hours.' - somewhere